website/src/_posts/2021-12-2.0-2.3.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Life after 2.0: new Audio plugin, Status Bar improvements" 
+date: 2021-12-15
+author: 
+  - aduh95
+  - murderlon
+  - arturi
+image: https://uppy.io/images/blog/2.0/uppy-2-0-cover.jpg
+published: true
+---
+
+After the 2.0 release we’ve been busy refactoring, updating dependencies, moving from npm to Yarn 3 and much-needed housekeeping. We’ve also released a new Audio plugin, improved Status Bar and Unsplash
+
+<!--more-->
+
+## Audio plugin
+
+Uppy now has a new `@uppy/audio` plugin for recording audio memos, messages and sounds. Check it out [on the demo page](https://uppy.io/examples/dashboard/).
+
+<video alt="Audio plugin demo" muted autoplay loop>
+  <source src="/images/blog/2.0-2.3/audio-demo.mp4" type="video/mp4">
+  Your browser does not support the video tag: https://uppy.io/images/blog/2.0-2.3/audio-demo.mp4
+</video>
+
+## Status bar error state improvements
```

Comment:
Personally, I find this not too significant of a change. I would put it further down at least